scripts/azureml-assets/azureml/assets/publish_utils.py

```diff
@@ -453,7 +454,7 @@ def update_asset_metadata(asset: AssetConfig, ml_client: MLClient, allow_no_op_u
                 tags = model_spec.get("tags", {})
                 properties = model_spec.get("properties", {})
 
-                tags = {k: util.util.resolve_from_file(model_config._append_to_file_path(v)) for k, v in tags.items()}
+                tags = {k: resolve_from_file_for_asset(asset, v) for k, v in tags.items()}
```

Comment:
Should the AssetConfig be used instead of model_config?

Reply:
I don't think it would matter too much?